packages/trace-viewer/src/ui/uiModeView.tsx

```diff
@@ -239,6 +253,21 @@ export const UIModeView: React.FC<{}> = ({
       setProgress(undefined);
   }, [testModel, isRunningTest]);
 
+  React.useEffect(() => {
+    if (!testModel)
+      return;
+
+    if (testModel.rootSuite.suites.length === 0) {
+      let prefix: string;
+      if (!configProjects || configProjects.length < 1)
+        prefix = 'No projects matched.';
+      else
+        prefix = `No projects matched. Available projects: ${configProjects.map(p => `"${p}"`).join(', ')}.`;
```

Comment:
Why is this a critical error? I should be able to check the project in the GUI and things are good?